src/config/paths.ts

```diff
@@ -8,6 +8,16 @@ export const buildItemPath = (id = ':itemId'): string => `${ITEMS_PATH}/${id}`;
 export const REDIRECT_PATH = '/redirect';
 export const MEMBER_PROFILE_PATH = '/profile';
 export const RECYCLE_BIN_PATH = '/recycle-bin';
-export const buildItemSettingsPath = (id = ':itemId'): string =>
-  `${ITEMS_PATH}/${id}/settings`;
 export const ITEM_ID_PARAMS = 'itemId';
+export const ITEM_SHARE_PATH = 'share';
+export const ITEM_PUBLISH_PATH = 'publish';
+export const ITEM_SETTINGS_PATH = 'settings';
+export const ITEM_INFORMATION_PATH = 'information';
+export const buildItemSettingsPath = (id = ':itemId'): string =>
+  `${ITEMS_PATH}/${id}/${ITEM_SETTINGS_PATH}`;
+export const buildItemSharePath = (id = ':itemId'): string =>
+  `${ITEMS_PATH}/${id}/${ITEM_SHARE_PATH}`;
+export const buildItemPublishPath = (id = ':itemId'): string =>
+  `${ITEMS_PATH}/${id}/${ITEM_PUBLISH_PATH}`;
+export const buildItemInformationPath = (id = ':itemId'): string =>
+  `${ITEMS_PATH}/${id}/${ITEM_INFORMATION_PATH}`;
```

Comment:
I think react-router-dom has some methods to resolve these paths depending on where they are created. look at [`useResolvedPath`](https://reactrouter.com/en/main/hooks/use-resolved-path) which allows to use a relative path that is then resolved to the absolute path depending where you are in the app.